northstar/src/runtime/error.rs

```diff
@@ -25,7 +25,10 @@ pub enum Error {
     /// The container is not known to the system
     #[error("Invalid container {0}")]
     InvalidContainer(Container),
-    /// The container cannot be started because it's already running
+    /// The container cannot be umounted because it is busy
```

Comment:
Comment does not match error. Maybe we can even remove these comments as the error message is pretty clear.